Makefile

```diff
@@ -13,7 +13,8 @@ GIT_BRANCH := $(shell git rev-parse --abbrev-ref HEAD 2>/dev/null)
 GIT_BRANCH_CLEAN := $(shell echo $(GIT_BRANCH) | sed -e "s/[^[:alnum:]]/-/g")
 RUNC_IMAGE := runc_dev$(if $(GIT_BRANCH_CLEAN),:$(GIT_BRANCH_CLEAN))
 PROJECT := github.com/opencontainers/runc
-BUILDTAGS ?= seccomp
+SELINUX_TAG := $(shell selinuxenabled 2>/dev/null && echo selinux)
+BUILDTAGS ?= seccomp $(SELINUX_TAG)
```

Comment:
Can we add apparmor using `aa-enabled` for consistency?

Reply:
Good idea; done